backend/cellguide/pipeline/metadata/__init__.py

```diff
@@ -0,0 +1,7 @@
+from backend.cellguide.pipeline.metadata.metadata_generator import generate_cellguide_card_metadata
+from backend.cellguide.pipeline.metadata.types import CellMetadata
+from backend.cellguide.pipeline.ontology_tree.tree_builder import OntologyTreeBuilder
+
+
+def run(ontology_tree: OntologyTreeBuilder) -> dict[str, CellMetadata]:
+    return generate_cellguide_card_metadata(ontology_tree.all_cell_type_ids_in_corpus)
```

Comment:
Can we add logging statements for the following:
 -  how many cell types were added out of how many total in corpus
 - which cell type term IDs in the corpus were filtered out because they were obsolete?
 - how many cell types had CL descriptions (this is a useful indicator to see the comprehensiveness of CL)